todo_app/app.py

```diff
@@ -1,27 +1,68 @@
-from flask import Flask, redirect, url_for, request
-from flask import render_template
+from flask import Flask, session, redirect, url_for, request
+from flask.templating import render_template
 
 from todo_app.flask_config import Config
 from todo_app.data.trello_items import get_items, add_item, complete_card
 
 
-app = Flask(__name__)
-app.config.from_object(Config())
+class ViewModel:
```

Comment:
Not an issue, but it's fairly standard to place classes in their own file, in order to prevent the amount of code in this one file from growing forever